Common/Input/InputState.h

```diff
@@ -78,24 +78,50 @@ enum {
 #define MAX_KEYQUEUESIZE 20
 #endif
 
-// Represents a single bindable key
+// Represents a bindable key. Although it can be bound to two keys - you can bind key combinations as keys.
 class KeyDef {
 public:
-	KeyDef() : deviceId(0), keyCode(0) {}
-	KeyDef(int devId, int k) : deviceId(devId), keyCode(k) {}
+	KeyDef() : deviceId(0), keyCode(0), deviceId2(0), keyCode2(0), dual(false) {}
+	KeyDef(int devId, int k) : deviceId(devId), keyCode(k), deviceId2(0), keyCode2(0), dual(false) {}
+	KeyDef(int devId, int k, int devId2, int k2) : deviceId(devId), keyCode(k), deviceId2(devId2), keyCode2(k2), dual(true) {}
 	int deviceId;
 	int keyCode;
 
+	bool dual;  // decides if deviceId2 and keyCode2 are valid.
+
+	int deviceId2;
+	int keyCode2;
```

Comment:
It kinda feels like these ought to be vectors instead, with the constructor pushing the single value for things like `hardcodedCancelKeys`.

Not saying we try to support it now, but it feels like if we need to redo any interfaces, it's better to design it so we could have three keys at once if we wanted.  I could see an argument for "ctrl + shift + space" being something.

-[Unknown]